exchanges/binance/binance.go

```diff
@@ -132,19 +135,20 @@ func (b *Binance) GetExchangeInfo() (ExchangeInfo, error) {
 //
 // symbol: string of currency pair
 // limit: returned limit amount
-func (b *Binance) GetOrderBook(symbol string, limit int64) (OrderBook, error) {
+// 获取交易深度
+func (b *Binance) GetOrderBook(obd OrderBookDataRequestParams) (OrderBook, error) {
 	orderbook, resp := OrderBook{}, OrderBookData{}
 
-	if err := b.CheckLimit(limit); err != nil {
-		return orderbook, err
-	}
-	if err := b.CheckSymbol(symbol); err != nil {
-		return orderbook, err
-	}
+	// if err := b.CheckLimit(limit); err != nil {
```

Comment:
Could we remove the code instead of commenting it out?

Reply:
I agree, in git we can always get the code back in case we would need it later.